test/els_completion_SUITE.erl

```diff
@@ -88,10 +88,6 @@ atom_completions(Config) ->
                 , kind             => ?COMPLETION_ITEM_KIND_MODULE
                 , label            => <<"code_navigation_types">>
                 }
-             , #{ insertTextFormat => ?INSERT_TEXT_FORMAT_PLAIN_TEXT
```

Comment:
I think this was an actual bug. We were including header files as completions for atoms. This was due to the fact we were storing modules and headers together. With the new implementation this is not the case anymore.